### PR TITLE
sdit-927 further adjudication migration tests and common code sharing

### PR DESCRIPTION
**Pull request by lbennett-moj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationsMigrationIntTest.kt [hunk 1]
```diff
@@ -103,10 +118,524 @@ class AdjudicationsMigrationIntTest : SqsIntegrationTestBase() {
...
+          .build()
+      }
+        .headers(setAuthorisation(roles = listOf("ROLE_MIGRATE_ADJUDICATIONS")))
+        .header("Content-Type", "application/json")
+        .exchange()
+        .expectStatus().isOk
+        .expectBody()
+        .jsonPath("$.size()").isEqualTo(2)
+        .jsonPath("$[0].migrationId").isEqualTo("2020-01-02T00:00:00")
+        .jsonPath("$[1].migrationId").isEqualTo("2020-01-01T00:00:00")
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationsMigrationIntTest.kt:353]: Just noticed the different order here - is that on purpose?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/appointments/AppointmentsMigrationIntTest.kt [hunk 2]
```diff
@@ -110,7 +114,8 @@ class AppointmentsMigrationIntTest : SqsIntegrationTestBase() {
       }
 
       // check filter matches what is passed in
-      nomisApi.verifyGetAppointmentsIdsCount(
+      nomisApi.verifyGetIdsCount(
+        url = "/appointments/ids",
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/appointments/AppointmentsMigrationIntTest.kt:118]: Isn't there a contsant for this?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/sentencing/SentencingMigrationIntTest.kt [hunk 3]
```diff
@@ -75,12 +80,13 @@ class SentencingMigrationIntTest : SqsIntegrationTestBase() {
...
     internal fun `will start processing pages of sentence adjustments`() {
-      nomisApi.stubGetSentenceAdjustmentsInitialCount(86)
+      nomisApi.stubGetInitialCount(NomisApiExtension.ADJUSTMENTS_ID_URL, 86) { adjustmentIdsPagedResponse(it) }
       nomisApi.stubMultipleGetAdjustmentIdCounts(totalElements = 86, pageSize = 10)
       nomisApi.stubMultipleGetSentenceAdjustments(1..86 step 2)
       nomisApi.stubMultipleGetKeyDateAdjustments(2..86 step 2)
-      mappingApi.stubAllNomisSentencingAdjustmentsMappingNotFound()
-      mappingApi.stubSentenceAdjustmentMappingCreate()
+      mappingApi.stubAllMappingsNotFound("/mapping/sentencing/adjustments/nomis-adjustment-category/SENTENCE/nomis-adjustment-id")
+      mappingApi.stubAllMappingsNotFound("/mapping/sentencing/adjustments/nomis-adjustment-category/KEY-DATE/nomis-adjustment-id")
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/sentencing/SentencingMigrationIntTest.kt:88]: No constants for these?

**Review by mikehalmamoj — DISMISSED** [review]

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationsMigrationIntTest.kt (hunk 1 above)
Comment (lbennett-moj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationsMigrationIntTest.kt:353]: this looks fine - migration time descending

**Pull request merged**